lib/improver/utilities/cube_manipulation.py

```diff
@@ -38,101 +38,40 @@
 import iris
 from iris.coords import AuxCoord, DimCoord
 from iris.exceptions import CoordinateNotFoundError
+
 from improver.utilities.cube_checker import (
     check_cube_coordinates, check_cube_not_float64, find_threshold_coordinate)
 
 
-def _associate_any_coordinate_with_master_coordinate(
```

Comment:
I'm happy all the changes in this file are due to moving code from cube_manipulation_new to cube_manipulation